.github/workflows/nightly-deps-test.yml

```diff
@@ -56,4 +56,4 @@ jobs:
       - run: cmake --preset ${{ matrix.variant.cmake-preset }}
       - run: cmake --build --preset build
       - run: ctest --preset test
-      - run: tox -e nightlydeps -vvv -- --hypothesis-profile=ci
```

Comment:
Instead of this, we could pass the `CI` env var in tox. That would make hypothesis use the ci profile and would be consistent with running it without tox.